latest_builds/matrix.yml

```diff
@@ -49,6 +49,21 @@ testing-ansible-12:
       dists:
         - noble
         - plucky
+testing-ansible-13:
+  github_branch: ansible-13
+  packages:
+    - name: ansible-core
+      version_specifier: "~=2.20.0a"
```

Comment:
ansible-core has beta releases, so shouldn't it be
```suggestion
      version_specifier: "~=2.20.0b"
```
?

Reply:
Pushed some updates... Let me know what you think @felixfontein & @mariolenz